protoc_plugin/lib/names.dart

```diff
@@ -80,6 +84,9 @@ String avoidInitialUnderscore(String input) {
   while (input.startsWith('_')) {
```

Comment:
Perhaps rename this function. Not sure I have a good name, but the current name becomes misleading.
Also update the comment

Reply:
Sorry that comment came too late. I see you already merged. Nevermind, it is so rare anyway.